apps/frontpage/lib/get-page.tsx

```diff
@@ -72,11 +72,14 @@ export const getPageData = async (
       li: MDX.List,
       pre: MDX.Pre,
       details: () => <details>Hello world</details>,
+      // eslint-disable-next-line react/jsx-pascal-case -- TODO: Not sure why this through an error.
```

Comment:
I suspect it wants it to be `<Mdx.Img />`.

Reply:
But that's what I did ... But the fact that I'm passing props makes it a problem.

Reply:
I really don't get it.